### PR TITLE
Apply dependencies update

### PR DESCRIPTION
**Pull request by jerome-benoit**
Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

## PR Checklist

- [x] Please add a description of your changes.
- [ ] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] If your changes contain any new feature please be sure to document it.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -28,8 +28,7 @@ module.exports = defineConfig({
     'plugin:import/warnings',
     'plugin:import/typescript',
     'plugin:promise/recommended',
-    'plugin:prettierx/standardx',
-    'plugin:prettierx/@typescript-eslint'
```
Comment (Shinigami92) [inline .eslintrc.js:32]: **question:** Why do you want to remove this? :eyes: 
I didn't found something in the changelog that points to that a removal is needed or anything. Could you explain?

Comment (jerome-benoit) [reply .eslintrc.js:32]: Read the changelog harder.

Comment (Shinigami92) [reply .eslintrc.js:32]: I don't understand
Haven't found anything here https://github.com/aMarCruz/eslint-plugin-prettierx/blob/master/CHANGELOG.md#0180---2021-04-27
And also the https://github.com/aMarCruz/eslint-plugin-prettierx/blob/master/README.md#supported-plugins doesn't explain something special about it

Help me and send me a link to the explanation, or I cannot approve this :shrug:

Comment (jerome-benoit) [reply .eslintrc.js:32]: https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4 

Presets removal is stated in the changelog: three presets only.

Comment (jerome-benoit) [reply .eslintrc.js:32]: https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4 

Presets removal is stated in the changelog: three presets only.

Comment (Shinigami92) [reply .eslintrc.js:32]: So you mean this: https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101

Really bad written by their changelogs... :confused:

**Review by Shinigami92 — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (jerome-benoit) [reply .eslintrc.js:32]: > So you mean this: [aMarCruz/eslint-plugin-prettierx@9af81e2#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101](https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101)
> 
> Really bad written by their changelogs... 😕

I've reported that issue a month ago. I think I will make a PR if the owner does not refine the documentation and the changelog.

**Pull request merged**